### PR TITLE
PAINTROID-259 Refactor tools.drawable package to Kotlin

### PR DESCRIPTION
**Pull request by CodeChamp-SS**
Refactored tools.drawable package to Kotlin.

https://jira.catrob.at/browse/PAINTROID-259

Files that were refactored:

1. DrawableFactory
2. DrawableShape
3. DrawableStyle
4. HeartDrawable
5. OvalDrawable
6. RectangleDrawable
7. ShapeDrawable
8. StarDrawable

### Your checklist for this pull request
Please review the [contributing guidelines](https://github.com/Catrobat/Paintroid/blob/develop/README.md) and [wiki pages](https://github.com/Catrobat/Catroid/wiki/) of this repository.

- [x] Include the name of the Jira ticket in the PR’s title
- [x] Include a summary of the changes plus the relevant context
- [x] Choose the proper base branch (*develop*)
- [x] Confirm that the changes follow the project’s coding guidelines
- [x] Verify that the changes generate no compiler or linter warnings
- [x] Perform a self-review of the changes
- [x] Verify to commit no other files than the intentionally changed ones
- [ ] Include reasonable and readable tests verifying the added or changed behavior
- [ ] Confirm that new and existing unit tests pass locally
- [x] Check that the commits’ message style matches the [project’s guideline](https://github.com/Catrobat/Catroid/wiki/Commit-Message-Guidelines)
- [x] Stick to the project’s gitflow workflow
- [x] Verify that your changes do not have any conflicts with the base branch
- [x] After the PR, verify that all CI checks have passed
- [x] Post a message in the *#paintroid* [Slack channel](https://catrobat.slack.com) and ask for a code reviewer

### CODE REVIEW

File: Paintroid/src/main/java/org/catrobat/paintroid/tools/drawable/DrawableFactory.kt [hunk 1]
```diff
@@ -16,16 +16,15 @@
...
-	public void draw(Canvas canvas, RectF shapeRect, Paint drawPaint) {
-		canvas.drawRect(shapeRect, drawPaint);
-	}
+class DrawableFactory {
+    fun createDrawable(shape: DrawableShape): ShapeDrawable {
+        return when (shape) {
+            DrawableShape.RECTANGLE -> RectangleDrawable()
+            DrawableShape.OVAL -> OvalDrawable()
+            DrawableShape.HEART -> HeartDrawable()
+            DrawableShape.STAR -> StarDrawable()
```
Comment (gPathpp) [inline Paintroid/src/main/java/org/catrobat/paintroid/tools/drawable/DrawableFactory.kt:27]: Nice 👍 
Obviously, the Kotlin compiler recognises that DrawableShape can only be those four, hence no IllegalArgumentException 😄 
Unfortunately, Github compares the wrong files (not your fault)

**Review by gPathpp — APPROVED** [review]
LGTM 👍

File: Paintroid/src/main/java/org/catrobat/paintroid/tools/drawable/DrawableFactory.kt (hunk 1 above)
Comment (CodeChamp-SS) [reply Paintroid/src/main/java/org/catrobat/paintroid/tools/drawable/DrawableFactory.kt:27]: yeah. Thanks! 😄

**Comment by wslany** [comment]
Thanks!

**Pull request merged**